lib/srv/app/common/audit.go

```diff
@@ -177,14 +177,14 @@ func (a *audit) OnRequest(ctx context.Context, sessionCtx *SessionContext, req *
 		Method:             req.Method,
 		Path:               req.URL.Path,
 		RawQuery:           req.URL.RawQuery,
-		StatusCode:         uint32(res.StatusCode),
+		StatusCode:         uint32(code),
```

Comment:
Instead of casting here, can we change the function argument? I think it makes more sense to accept the argument of expected type and cast when a function is called when needed.

Reply:
Good suggestion. I wonder why golang's `ResponseWriter` takes an int status code anyway? Shouldn't status > 0 always?